test/axios_vcr_test.js

```diff
@@ -85,38 +86,39 @@ describe('Axios VCR', function() {
       check that the response is the same as the cassette file.
     */
     it('skips remote calls', function(done) {
-      var path = './test/static_fixtures/posts.json'
-      assert(fileExists(path))
+      var fixturePrefix = process.env.FIXTURE_PREFIX || ''
+      var cassettePath = path.join('./test', fixturePrefix, 'static_fixtures', 'posts.json')
+      assert(fileExists(cassettePath))
 
       var url = 'http://something.com/unexisting'
-      VCR.mountCassette(path)
+      VCR.mountCassette(cassettePath)
 
       axios.get(url).then(function(res) {
-        getFixture(path, res.config).then(function(fixture) {
+        getFixture(cassettePath, res.config).then(function(fixture) {
           assert.deepEqual(fixture.originalResponseData, _.omit(res, 'fixture'))
           done()
 
-          VCR.ejectCassette(path)
-        }).catch(err => { console.log(err); done() })
```

Comment:
why would you remove the `catch` call here?
we want to make sure tests don't hang if they fail.